verticapy/core/vdataframe/_text.py

```diff
@@ -101,6 +101,87 @@ def regexp(
         -------
         vDataFrame
             self
+
+        Examples
+        ---------
+
+        Let's begin by importing `VerticaPy`.
+
+        .. ipython:: python
+
+            import verticapy as vp
+
+
+        Let's generate a small dataset using the following data:
+
+        .. ipython:: python
+
+            data = vp.vDataFrame(
+                            {
+                                "rollno": ['1', '2', '3', '4'],
+                                "subjects": ['English, Math', 'English, Math, Computer', 'Math, Computer, Science', 'Math, Science'],
+                            })
+
+        Let's retrieve the second subject.
+
+        .. code-block:: python
+
+            data.regexp(column = "subjects",
+                                 pattern = "[^,]+",
+                                 method = "substr",
+                                 occurrence = 2,
+                                 name = "subject_2").select(["subjects",
+                                                          "subject_2"])
+
+
+        .. ipython:: python
+            :suppress:
+
+            res = data.regexp(column = "subjects",
+                                 pattern = "[^,]+",
+                                 method = "substr",
+                                 occurrence = 2,
+                                 name = "subject_2").select(["subjects",
+                                                          "subject_2"])
+            html_file = open("figures/core_vDataFrame_text_regex1.html", "w")
+            html_file.write(res._repr_html_())
+            html_file.close()
+
+        .. raw:: html
+            :file: ../../../figures/core_vDataFrame_text_regex1.html
```

Comment:
This path should be 
SPHINX_DIRECTORY/figures/core....

The same change can be made throughout the docuement